bundle/manifests/openshift-ai-toolkit.clusterserviceversion.yaml

```diff
@@ -172,7 +182,7 @@ spec:
                 - --health-probe-bind-address=:8081
                 command:
                 - /manager
-                image: quay.io/openshift-ai-toolkit/openshift-ai-toolkit-operator-controller:latest
+                image: quay.io/dbhagava/openshift-ai-toolkit-operator-controller:latest
```

Comment:
this is not required. Changes to push to personal quay account is highly discouraged